libs/blocks/aside/aside.js

```diff
@@ -46,32 +51,115 @@ function decorateStaticLinks(el) {
   textLinks.forEach((link) => { link.classList.add('static'); });
 }
 
+function decorateModalImage(el, lnkImg) {
+  if (el.classList.contains('split')) lnkImg.closest('div').classList.add('has-modal');
+  lnkImg.classList.add('play-btn');
+  const playIcon = createTag('div', { class: 'play-icon-container', 'aria-label': 'play' }, PLAY_ICON);
+  const playCircle = createTag('div', { class: 'play-btn-circle', 'aria-label': 'play' }, playIcon);
+  const playContainer = createTag('div', { class: 'play-container', 'aria-label': 'play' });
+  lnkImg.parentNode.appendChild(lnkImg.querySelector('picture'));
+  lnkImg.parentNode.appendChild(playContainer);
+  lnkImg.appendChild(playCircle);
```

Comment:
I think this functionality, to add the play button over an image, should be universal to images, not tied directly to Asides. 
This probably should go in [utils.js where image linking is set up](https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L443). 

@chrischrischris @auniverseaway thoughts?

Reply:
I agree. The video play should be its own universal PR. Also, In this same vein, there's more suggestions below in the resolved conversation about using shared functions I think we should re-consider.

Reply:
@ryanmparrish @rgclayton We have removed the play button css and implementation from this PR and will be taking it up as a part of generic play button addition ticket https://jira.corp.adobe.com/browse/MWPW-127223